src/commands/show.ts

```diff
@@ -79,4 +81,83 @@ export const show = async (): Promise<void> => {
   console.log("  Total Staked Value (SOL) ", tvlStaked.toLocaleString())
   console.log("  Total Liquidity-Pool (SOL) ", tvlLiquidity.toLocaleString())
   console.log("  TVL (SOL) ", (tvlStaked + tvlLiquidity).toLocaleString())
+
+  if ('list' in options) {
+    console.log()
+    console.log("  Validator_manager_authority", state.validatorSystem.managerAuthority.toBase58())
+    console.log(`  Stake list account: ${state.stakeSystem.stakeList.account} with ${state.stakeSystem.stakeList.count}/${"?"} stakes`)
+    console.log("-----------------")
+    console.log("-- Validators ---")
+    console.log(`  Total staked: ${MarinadeUtils.lamportsToSol(state.validatorSystem.totalActiveBalance)} SOL`)
+    console.log(`  List account: ${state.validatorSystem.validatorList.account} with ${state.validatorSystem.validatorList.count}/${"?"} validators`)
+    console.log("-------------------------------------------------------------")
+
+    const epochInfo = await marinadeState.epochInfo()
+    const validatorAccounts = await marinadeState.getValidatorRecords()
+    const stakeInfos = await marinadeState.getStakeInfos()
+
+    let totalStaked = new BN(0)
+    let totalStakedFullyActivated = new BN(0)
+
+    validatorAccounts
+        .forEach((validator, validatorIndex) => {
+      if (validator.activeBalance.toNumber() <= 0) return
+
+      const validatorStakes:StakeInfo[] = stakeInfos
+          .filter(stakeInfo => stakeInfo.stake.Stake?.stake.delegation)
+          .filter(stakeInfo => U64_MAX.eq(stakeInfo.stake.Stake?.stake.delegation.deactivationEpoch as BN))
+          .filter(stakeInfo => stakeInfo.stake.Stake?.stake.delegation.voterPubkey.toBase58() == validator.validatorAccount.toBase58())
+      console.log(`${validatorIndex+1}) Validator ${validator.validatorAccount.toBase58()}`
+          + `, marinade-staked ${MarinadeUtils.lamportsToSol(validator.activeBalance).toFixed(2)} SOL`
+          + `, score-pct: ${(validator.score * 100/ state.validatorSystem.totalValidatorScore).toFixed(4)}%, ${validatorStakes.length} stake-accounts`)
+
+      for (const [index, stakeInfo] of validatorStakes.entries()) {
+
+        let delegation = stakeInfo.stake.Stake?.stake.delegation as Delegation
+        let meta = stakeInfo.stake.Stake?.meta as Meta
+
+        let extraBalance = MarinadeUtils.lamportsToSol(
+            stakeInfo.balance
+                .add(delegation.stake.neg())
```

Comment:
maybe I'm wrong bu shouldn't be `.sub(` here?
is there a reason to use `.add(... .neg())`

Reply:
It's my fault for missing function sub